package.json

```diff
@@ -33,6 +33,7 @@
     "@chakra-ui/react": "^2.4.1",
     "@contentful/f36-icons": "^4.23.2",
     "@contentful/f36-tokens": "^4.0.1",
+    "@contentful/live-preview": "^1.15.0",
```

Comment:
If it is not a big change for you, would you mind updating it to 1.16 and use the optimized react implementation? :) 
Otherwise we can also do it in a follow up ticket

Reply:
Will do!